app/utils/transactionFlows/configureBaker.ts

```diff
@@ -269,7 +269,7 @@ export const convertToBakerTransaction = (
         }).every(not(isDefined))
     ) {
         throw new Error(
-            'Trying to submit a transaction without any changes to the existing baker configuration of an account.'
+            'Trying to submit a transaction without any changes to the existing validation configuration of an account.'
```

Comment:
Should this be validator instead of validation? As baker -> validator. Or should it have been baking previously?

Reply:
Yeah should be validator, both kinda sounds wrong though. I'll change it, doesn't really matter (This is not the popup message shown at the end of the flow, that one says "validator configuration")